src/ontology/mondo-edit.obo

```diff
@@ -420511,24 +420513,21 @@ is_a: MONDO:0005046 {source="Orphanet:310050"} ! immune system disorder
 
 [Term]
 id: MONDO:0017770
-name: Robinow-like syndrome
-def: "Robinow-like syndrome is characterized by the association of the clinical features present in Robinow syndrome (short stature, mesomelic brachymelia, macrocephaly, and hypoplastic genitalia), with anterior chamber cleavage anomalies. It has been described in two sisters and is transmitted as an autosomal recessive trait." [Orphanet:3105]
+name: obsolete Robinow-like syndrome
+def: "OBSOLETE. Robinow-like syndrome is characterized by the association of the clinical features present in Robinow syndrome (short stature, mesomelic brachymelia, macrocephaly, and hypoplastic genitalia), with anterior chamber cleavage anomalies. It has been described in two sisters and is transmitted as an autosomal recessive trait." [Orphanet:3105]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -
-subset: gard_rare {source="MONDO:GARD"}
-subset: obsoletion_candidate
 subset: ordo_malformation_syndrome {source="Orphanet:3105"}
 subset: otar {source="MONDO:OTAR"}
-subset: rare
 synonym: "Saal-Greenstein syndrome" EXACT []
 xref: ICD10CM:Q87.1 {source="Orphanet:3105/attributed", source="Orphanet:3105/ntbt", source="Orphanet:3105"}
-xref: MEDGEN:928625 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:3105 {source="MONDO:equivalentObsolete"}
-xref: SCTID:721905000 {source="MONDO:equivalentTo"}
-xref: UMLS:C4302956 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:928625"}
-is_a: MONDO:0015160 {source="Orphanet:3105"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
-is_a: MONDO:0021147 {source="Orphanet:3105", source="Orphanet:3105/inferred"} ! disorder of development or morphogenesis
+xref: MEDGEN:928625 {source="MONDO:obsoleteEquivalent", source="MONDO:MEDGEN"}
+xref: Orphanet:3105 {source="MONDO:obsoleteEquivalent"}
```

Comment:
It looks like this was also obsoleted in Orphanet.
Therefore the source should be "MONDO:obsoleteEquivalentObsolete"